src/script/conversation/ConversationRepository.ts

```diff
@@ -371,24 +371,6 @@ export class ConversationRepository {
     }
   };
 
-  /**
-   * Remove obsolete conversations locally.
-   */
-  cleanupConversations(): void {
-    this.conversationState.conversations().forEach(conversationEntity => {
-      if (
-        conversationEntity.isGroup() &&
-        conversationEntity.is_cleared() &&
-        conversationEntity.removed_from_conversation()
-      ) {
-        this.conversationService.deleteConversationFromDb(conversationEntity.id);
-        this.deleteConversationFromRepository(conversationEntity);
-      }
-    });
-
-    this.cleanupEphemeralMessages();
```

Comment:
We don't want cleared conversations we're not a member of to be automatically deleted when app is loaded.